### PR TITLE
Initial Zim manifest file

### PR DESCRIPTION
**Pull request by bertinatto**

### CODE REVIEW

File: org.zimwiki.Zim.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "org.zim.Zim",
```
Comment (nedrichards) [inline org.zimwiki.Zim.json]: The upstream app ID is org.zim-wiki.Zim and [there's a PR ](https://github.com/jaap-karssenberg/zim-desktop-wiki/pull/307)to change to org.zim_wiki.Zim which is flatpak compatible.

All your exports (e.g. desktop files, icons and appdata etc.) have to have the same app-id naming.

**Comment by cheese** [comment]
Yes, I am waiting for upstream to decide the new appid. But upstream author is working on gtk3/python3 porting. After the porting, flatpak building will also turn much simpler.

File: org.zimwiki.Zim.json (hunk 1 above)
Comment (bertinatto) [reply org.zimwiki.Zim.json]: Thanks for pointing this out!

**Comment by bertinatto** [comment]
Great, thanks for the info!

Just for the context, on Feb 9 I sent an email to @jaap-karssenberg asking if he wanted to have a Flatpak for Zim and offering my help for it. He kindly replied saying that he wouldn't have objections if I submitted the Flatpak, as he's Ubuntu-based and prefers to use the PPA he created for Zim.

**Comment by cheese** [comment]
Yes, the blocking issue is upstream should change the appid first. I submitted a pr to change appid to "org.zim_wiki.Zim".
https://github.com/jaap-karssenberg/zim-desktop-wiki/pull/307

**Comment by cheese** [comment]
Submitted: https://github.com/bertinatto/flathub/pull/1

**Comment by bertinatto** [comment]
Cool, thanks @cheese!

I merged your changes into my branch, but feel free to close this PR and open a separate one if you think it's better.

**Comment by cheese** [comment]
I would like to help maintaining this package as my first flatpak.

**Comment by nedrichards** [comment]
That's awesome! We'll make sure you're all included.

**Comment by nedrichards** [comment]
org.zim_wiki.Zim

**Comment by nedrichards** [comment]
bot, build org.zim_wiki.Zim

File: org.zim_wiki.Zim.json [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                    "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                }
+            ]
+        },
+        {
+            "name": "pygtk",
```
Comment (nedrichards) [inline org.zim_wiki.Zim.json]: You need to copy the reconfiguration from here in order to make this build on aarch64 https://github.com/flathub/org.gimp.GIMP/blob/1618d23546fd49d52bd8a3704ebdd4f7e323c1c6/org.gimp.GIMP.base-modules.json#L77

**Comment by nedrichards** [comment]
bot, build org.zim_wiki.Zim

**Comment by nedrichards** [comment]
Next one, gtksourceview on aarch64! https://flathub.org/builds/#/builders/2/builds/30 same diagnosis.

**Pull request closed without merging**